lib/node/runner/checker.go

```diff
@@ -662,6 +661,11 @@ func finishBallot(st *storage.LevelDBBackend, b ballot.Ballot, transactionPool *
 		return nil, err
 	}
 
+	if err = finishDelayedOperations(blk, transactionPool.Pending, st); err != nil {
```

Comment:
@Geod24 I think a delayed operation is not included in ballot.
It was inserted into pending pool when unfreezing request comes.
And it is executed when reached proper blockheight.
Of course, the unfreezing request was saved.
But should not the proposer add delayed operation to ballot again?

Reply:
@Geod24 IMO, delayed operation is actual operation which affects balance of accounts.
And I think that it is also agreed through voting among the nodes.

Reply:
A delayed operation was agreed on at the time it entered the blockchain, so when it is actually applied, it does not need to be re-voted on.